tool/nixvirt.py

```diff
@@ -121,19 +121,19 @@ def _descriptionXMLText(self,lvobj):
         # VIR_NETWORK_XML_INACTIVE
         return lvobj.XMLDesc(flags=1)
     def _getDependents(self,obj):
-        networknames = [str(name) for name in obj.descriptionXMLETree().xpath("/network/name")]
-        bridgenames = [str(name) for name in obj.descriptionXMLETree().xpath("/network/bridge/@name")]
+        networknames = [name.text for name in obj.descriptionXMLETree().xpath("/network/name")]
+        bridgenames = [name.text for name in obj.descriptionXMLETree().xpath("/network/bridge/@name")]
```

Comment:
Is it actually needed here? The I thought the @selector returned a string already as it's the contents of the attribute vs the regular selector which is returning an element and so the text needs to be extracted.

Reply:
I think so? https://lxml.de/xpathxslt.html#xpath-return-values
It says it returns "smart strings", but not strings.